src/preferencesdialog.ui

```diff
@@ -117,6 +117,13 @@
          </property>
         </widget>
        </item>
+       <item row="9" column="0" colspan="2">
+        <widget class="QCheckBox" name="useTrashBox">
+         <property name="text">
+          <string>Use system Trash (and don't prompt)</string>
```

Comment:
Very good! Please just change "don't" to "do not".

Reply:
Done.